src/js/widgets/views/toolbardrawV.js

```diff
@@ -275,20 +277,34 @@
 
 						// add import and export buttons
 						'<input id="fileDialogAnno' + mapid + '" type="file" accept="application/json" data-bind="event: { change: importClick }" tabindex="-1"></input>' +
-						'<button class="gcviz-draw-imp" tabindex="0" data-bind="buttonBlur, click: launchDialog, attr: { alt: tpImport }"></button>' +
+						'<button class="gcviz-draw-imp" tabindex="0" data-bind="buttonBlur, click: launchDialog, attr: { alt: tpImport } ,tooltip: { content: tpImport } "></button>' +
 						'<div class="gcviz-tbseparator"></div>' +
-						'<button class="gcviz-draw-exp" tabindex="0" data-bind="buttonBlur, click: exportClick, attr: { alt: tpExport }, enable: isGraphics"></button>' +
+						'<button class="gcviz-draw-exp" tabindex="0" data-bind="buttonBlur, click: dialogFileNameClick, attr: { alt: tpExport }, enable: isGraphics ,tooltip: { content: tpExport }"></button>' +
+		//		'<button class="gcviz-draw-exp" tabindex="0" data-bind="buttonBlur, click: exportClick, attr: { alt: //tpExport }, enable: isGraphics ,tooltip: { content: tpExport }"></button>' +
 
 					// close the span for buttons
 					'</div>' +
 
 					// add text
 					'<div class="span8">' +
 						'<span class="gcviz-drawspanlabel" data-bind="text: lblImpExp"></span>' +
-					'</div>' +
-
+					'</div>' + 					
 				// close row
-				'</div>';
+                   '</div>'+ 
+         //          '</div>;' ;
+				// added pw.no 2016
+				'<div data-bind="uiDialog: { title: tpOutputFileTitle, width: 550, ok: dialogFileNameOk, cancel: dialogFileNameCancel, close: dialogTextClose, openDialog: \'isTextDialogOpenpw\' }">' +
+							'<div id="gcviz-draw-inputbox">' +
+								'<form><fieldset>' +
+									'<input id="gcviz-textvalue" class="gcviz-draw-textinput text ui-widget-content ui-corner-all" data-bind="value: drawTextValuepw, valueUpdate: \'input\', returnKey: dialogFileNameEnterpw, attr: { alt: tpOutputFileTitle  }"/>' +
+									'<div style="clear: both"></div><span data-bind="text:tpOutputFileText"></span>'+
+								'</fieldset></form>' +
+							'</div>' +
+			//			'</div>'
+						'</div>';
+							
+				 
+				
 			}
```

Comment:
This seems to be related witht he other PR you sent. This is not tooltip related. Can you transfer this code into the other PR?